### ISSUE TITLE
Adding delsms command

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#85]
```
Adding delsms command.
delsms:id:[id]
delsms:contact:[contact]
delsms:all
delsms:sent

The syntax will be more complex on purpose.
```

Original issue reported on code.google.com by `Florent....@gmail.com` on 7 Feb 2011 at 8:08

**Comment by GoogleCodeExporter** [comment]
```
delsms:id not implemented because user don't have access to the sms id :)
```

Original comment by `Florent....@gmail.com` on 7 Feb 2011 at 11:13
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9.1
```

Original comment by `Florent....@gmail.com` on 13 Feb 2011 at 12:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
